pipelines/S1/queries.log

```diff
@@ -44,43 +45,146 @@ SELECT DISTINCT verbrauchskategorie, verbrauchskategorie_beschreibung FROM media
 Query Nr. 3
 -- Wie viel zahle ich in CHF für 1000 kWh bei einer 2-Zimmer-Wohnung im Kanton Bern im Jahr 2017?
 SELECT T.verbrauchskategorie, T.verbrauchskategorie_beschreibung, T.energieprodukt, T.mittlerer_preis_rappen_pro_kwh * 1000 / 100 as preis_franken_fur_1000_kwh_bern_2017_2_zimmerwohnung FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Bern%'     AND S.canton=TRUE     AND T.jahr=2017     AND T.verbrauchskategorie_beschreibung LIKE '%2-zimmerwohnung%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr >= 1000;
```

Comment:
Here the query was not changed, but result looks unexpected.

Reply:
This was because the `LIKE` function is case-insensitive in SQLite, but case sensitive in postgreSQL :(